### DISCUSSION TITLE
Why are teams required?

### DISCUSSION THREAD

**Discussion by AhiVT** [evroon/bracket#580] [Q&A] [answered]
I want to create a tournament where players compete 1-on-1, so no teams exist. Teams are a required field on the demo version of Bracket, so if I wanted to create a tournament I’d have a lot of tedious, repetitive and unnecessary work to place each player on their own team.

**Comment by evroon** [comment, marked as answer]
@AhiVT 
You can just create teams that have no players, and then give the team the name of the player it represents, right?

**Reply by AhiVT** [reply]
I wouldn’t be able to track player ELO  if I did so. (I understand that as far as I know ELO is a per-tournament, not a per-club statistic #581)

**Reply by evroon** [reply]
Oh ELO is calculated per team, it's just not shown in the admin overview of teams. There is a dashboard page that shows it:

`https://www.bracketapp.nl/tournaments/<id>/dashboard/standings`

**Reply by AhiVT** [reply]
Thanks for the information @evroon! I’m concerned that while your initial comment works great as a workaround, wouldn’t it be counterintuitive to new users? For now I’ll mark it as an answer until new functionality is added.
